.eslintrc.json

```diff
@@ -24,6 +24,7 @@
     "prettier"
   ],
   "rules": {
+    "import/prefer-default-export": "off",
```

Comment:
Why this? You are using named export and named import for `sendMail`, but it can easily be default export

Reply:
default is used when we have single export which is what we have now, but since we have created a helper we might add more methods to the file, so instead of getting an object and then fetching a key from it, i directly want to call all the keys of the object.